hugo/content/species/meganyctiphanes_norvegica/_index.md

```diff
@@ -35,12 +35,16 @@ The *Meganyctiphanes norvegica* species of krill is a macrozooplankton crustacea
 
 The tolerance of *M. norvegica* to a broader range of temperatures make it interesting for studies on ecological adaptation to the warming of the oceans. A challenge with genomic analysis of krill species is their huge genome size (19 Gb for *M. norvegica*) (Choquet et al. 2023; Jeffery, 2012; Unneberg et al., 2024). Genetic variation has been found to be widespread but divergent among specimens collected thoughout their geographic distribution, including polygeneic variants indicating adaptation potential to changes in light and temperature (Unneberg et al., 2024).
 
-#### Genome reference
+#### How to cite
 
 If you use the data presented in the genome portal from this species in your research, please cite the original publication:
 
 - <p> Unneberg, P., Larsson, M., Olsson, A., Wallerman, O., Petri, A., Bunikis, I., Vinnere Pettersson, O., Papetti, C., Gislason, A., Glenner, H., Cartes, J. E., Blanco-Bercial, L., Eriksen, E., Meyer, B., & Wallberg, A. (2024). Ecological genomics in the Northern krill uncovers loci for local adaptation across ocean basins. Nature Communications, 15(1), 6297. <a href="https://doi.org/10.1038/s41467-024-50239-7"> https://doi.org/10.1038/s41467-024-50239-7</a></p>
 
+If you have used the pages for this species in the Genome Portal, please refer to it in-text as: "The *Meganyctiphanes norvegica* entry in the Swedish Reference Genome Portal (access date)." and use the following for the bibliography:
```

Comment:
In a later iteration, I would suggest moving this snippet to the template to avoid duplication. We could also thereby include the current version and date (but that may be double edged)

Reply:
Right! I put an (now outdated) version of this in the template in [b44928c](https://github.com/ScilifelabDataCentre/genome-portal/pull/55/commits/b44928c94546fd7595e6c610a83c824a48d96e94). But an update is due, thanks for reminding me.

Reply:
I was ambiguous sorry: I meat in the HTML template where we could profit from `{{ $siteVersion }}` and `{{ .Date }}` type of things

Reply:
Ah yes, I like the idea. I thought of something similar myself. This could perhaps be related to the future investigation in EB-268 about hosting .bib/.ris citation files to improve UX.